src/JBArbitrumSucker.sol

```diff
@@ -221,29 +234,35 @@ contract JBArbitrumSucker is JBSucker, IJBArbitrumSucker {
             // Perform the ERC-20 bridge transfer.
             // slither-disable-start out-of-order-retryable
             // slither-disable-next-line calls-loop,unused-return
-            IArbL1GatewayRouter(address(GATEWAYROUTER)).outboundTransferCustomRefund{value: transportPayment}({
+            IArbL1GatewayRouter(address(GATEWAYROUTER)).outboundTransferCustomRefund{value: tokenTransportCost}({
                 token: token,
                 refundTo: msg.sender,
                 to: peer(),
                 amount: amount,
-                maxGas: MESSENGER_BASE_GAS_LIMIT,
+                maxGas: remoteToken.minGas,
```

Comment:
since there is headroom / potential leftover for tokeTransportCost, should there also be some here?